natlas-server/app/scope/scope.py

```diff
@@ -104,39 +106,24 @@ def update(self):
 		self.updateScanManager()
 		log("ScopeManager Updated")
 
-	def address_in_collection(self, targetAddr, networkCollection):
-		""" Take in a collection of networks, identify if the target address is in one of those networks """
-		inCollection = False
-		for network in networkCollection:
-			# TODO this eventually needs to be upgraded to support IPv6
-			if str(network).endswith('/32') and targetAddr == ipaddress.IPv4Address(str(network).split('/32')[0]):
-				inCollection = True
-			if targetAddr in network:
-				inCollection = True
-		return inCollection
-
 	def isAcceptableTarget(self, target):
 		# Ensure it's a valid IPv4Address
 		try:
 			# TODO this eventually needs to be upgraded to support IPv6
-			targetAddr = ipaddress.IPv4Address(target)
-		except ipaddress.AddressValueError:
+			targetAddr = IPAddress(target)
```

Comment:
Don't assign this to a return value if we don't use it anywhere else in the code. LGTM complains about this, and it's not really good practice to create a bunch of unused variables. This Try/Except block purely serves to validate that it's a valid IP address and we never use `targetAddr` afterwards. You can just:

```
try:
    IPAddress(target)
except AddrFormatError:
    return False
```